configs/_base_/models/dpt_vit-l16.py

```diff
@@ -0,0 +1,31 @@
+norm_cfg = dict(type='SyncBN', requires_grad=True)
+model = dict(
+    type='EncoderDecoder',
+    pretrained='pretrain/vit-l_timm.pth', # noqa
```

Comment:
Add checkpoint converter to `README.md` according to other transformer models (e.g. SegFormer).